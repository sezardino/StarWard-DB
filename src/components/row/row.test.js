import React from 'react';
import {shallow} from 'enzyme';

import Row from './row';

describe('Component Row', () => {
	const leftContent = 'Hello';
	const rightContent = 'World';

	it('Component rendered correctly', () => {
		const row = shallow(<Row left={leftContent} right={rightContent} />);
		expect(row.debug()).toMatchSnapshot();
	});
});
