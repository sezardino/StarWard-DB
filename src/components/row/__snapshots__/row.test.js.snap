// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component Row Component rendered correctly 1`] = `
"<div className=\\"row mb2\\">
  <div className=\\"col-md-6\\">
    Hello
  </div>
  <div className=\\"col-md-6\\">
    World
  </div>
</div>"
`;
